collectors/collectors.go

```diff
@@ -37,6 +37,13 @@ var (
 		[]string{"resource"},
 	)
 
+	ResourcesScrapedTotalMetric = prometheus.NewCounter(
```

Comment:
`ResourcesScrapedTotalMetric` should be registered in `main.go` just like what we do for `ScrapeErrorTotalMetric`. 

FYI: https://github.com/kubernetes/kube-state-metrics/blob/master/main.go#L248-L253

Reply:
Yes, it's here - https://github.com/kubernetes/kube-state-metrics/pull/427/files#diff-7ddfb3e035b42cd70649cc33393fe32cR250